src/Fable.Core/Fable.Core.JsInterop.fs

```diff
@@ -70,6 +70,10 @@ let jsTypeof (x: obj): string = jsNative
 [<Emit("$0 instanceof $1")>]
 let jsInstanceof (x: obj) (cons: obj): bool = jsNative
 
+/// Check if object is callable, i.e a function.
+[<Emit("typeof $0 == \"function\"")>]
+let inline callable(x: obj) = jsNative
```

Comment:
Ok, the problem was that it was used on `Array.fs` and I cannot use `jsTypeof` since it needs to work with Python as well.  I can try to move it to `Native.fs` instead. https://github.com/fable-compiler/Fable/pull/2345/files#diff-018d708b70b3410fb1a2c255b3bc655eee1aa0936b3b55da998b7aec9b2cd900L542